R-package/src/cmake/modules/FindLibR.cmake

```diff
@@ -174,17 +174,19 @@ execute_process(
   OUTPUT_VARIABLE LIBR_LIB_DIR
 )
 
+set(LIBR_HOME ${LIBR_HOME} CACHE PATH "R home directory")
+set(LIBR_EXECUTABLE ${LIBR_EXECUTABLE} CACHE PATH "R executable")
+set(LIBR_INCLUDE_DIRS ${LIBR_INCLUDE_DIRS} CACHE PATH "R include directory")
+set(LIBR_LIB_DIR ${LIBR_LIB_DIR} CACHE PATH "R shared libraries directory")
+
 # look for the core R library
 find_library(
   LIBR_CORE_LIBRARY
-  NAMES R
-  HINTS "${CMAKE_CURRENT_BINARY_DIR}" "${LIBR_LIB_DIR}" "${LIBR_HOME}/bin" "${LIBR_LIBRARIES}"
+  NAMES R R.dll
+  HINTS "${CMAKE_CURRENT_BINARY_DIR}" "${LIBR_HOME}/lib" "${LIBR_HOME}/bin/${R_ARCH}" "${LIBR_HOME}/bin" "${LIBR_LIBRARIES}"
+  PATHS "${LIBR_HOME}/bin/${R_ARCH}/R.dll"
```

Comment:
Hmm, is `PATHS` here really useful?.. According to the API,
> Additional search locations can be specified after the PATHS argument. 
   https://cmake.org/cmake/help/v3.0/command/find_library.html

and WIKI
> PATHS is used to provide additional search paths for CMake to look into and it should not generally be defined for things other than pkg-config (CMake has its built-in defaults and more can be added as required by various config variables). If you are not using it, leave out the entire section.
https://gitlab.kitware.com/cmake/community/-/wikis/doc/tutorials/How-To-Find-Libraries#finding-files

In addition, I thought that `PATHS` should contain folders, not exact paths.

Reply:
Ah yeah you are probably right. I originally used it because of the line in the docs that says "PATHS should be used for hard-coded guesses". I'll remove it, good catch!